### ISSUE TITLE
Where is the example from the Dash User's Guide?

### ISSUE THREAD

**Issue by marfel** [plotly/dash-table#263]
I apologize if this is not the right place to post this, but the Component Libraries example in the Dash User's Guide https://dash.plot.ly/gallery links directly to this repository and the data-table documentation, but not to the source code of the nice looking example (claimed to be written in < 100 lines of code). Where can I find this code? It looks quite like what I need right now.

**Comment by Marc-Andre-Rivet** [comment]
@marfel I'm not sure where the gallery example comes from (@chriddyp ?) but here is are two similar examples that cover most of what can be seen in the gallery.

For the graph / table interaction
https://dash.plot.ly/datatable/interactivity
(Source: https://github.com/plotly/dash-docs/blob/master/tutorial/examples/table/interactivity_connected_to_graph.py)

For styling
https://dash.plot.ly/datatable/style
(Source: https://github.com/plotly/dash-docs/blob/master/tutorial/table/styling_chapter.py)

The table's selected rows styling would have been done with a custom css (see the table's css prop) as we do not currently expose a standardized API for styling selected rows.

**Comment by Marc-Andre-Rivet** [comment]
You're right. Something is off about that table and it does seem to be built on the prototype.

1. You can get the checkboxes by adding ```row_selectable: 'multi'```

2. Operators are required in the current version of the table -- we've discussed adding those but as you point out in your comment in the filtering epic, it's not there yet.

While there is a desire for more powerful features related to filtering, you are right that making it more user friendly in the short term would be very helpful. Created an issue for the no-operator-filter here https://github.com/plotly/dash-table/issues/277
